Formula/httest.rb

```diff
@@ -16,17 +16,28 @@ class Httest < Formula
   depends_on "openssl"
   depends_on "pcre"
   depends_on "lua"
+  depends_on "nghttp2"
+  depends_on "spidermonkey"
 
   def install
     # Fix "fatal error: 'pcre/pcre.h' file not found"
     # Reported 9 Mar 2017 https://sourceforge.net/p/htt/tickets/4/
     (buildpath/"brew_include").install_symlink Formula["pcre"].opt_include => "pcre"
     ENV.prepend "CPPFLAGS", "-I#{buildpath}/brew_include"
 
+    # Fix "ld: file not found: /usr/lib/system/libsystem_darwin.dylib" for lxml
+    ENV["SDKROOT"] = MacOS.sdk_path if MacOS.version == :sierra || MacOS.version == :el_capitan
```

Comment:
@ilovezfs, would it make more sense to write as `if MacOS.version < :high_sierra`?

Reply:
nope this is fine

Reply:
you should use block-if though to keep the line length down